tests/unit/test_legal_search.py

```diff
@@ -69,7 +68,43 @@ def test_search_advisory_opinions(self, load_legal_search_results):
         assert response.status_code == 200
 
         load_legal_search_results.assert_called_once_with('in kind donation',
-            'advisory_opinions', None, None, None, None, None, None, 0, None, offset=0)
+            'advisory_opinions', None, None, None, None, None, None, 0, None, None, offset=0)
+
+    @mock.patch.object(api_caller, '_call_api')
+    def test_api_invoked_correctly_for_ao(self, _call_api):
+        _call_api.return_value = {}
+        self.app.get(
+            '/legal/search/advisory-opinions/',
+            data={
+                'search': 'in kind donation',
+                'search_type': 'advisory_opinions'
+            }
+        )
+        _call_api.assert_called_once_with('legal',
+                                          'search',
+                                          from_hit=0,
+                                          hits_returned=20,
+                                          q='in kind donation',
+                                          type='advisory_opinions')
+
+    @mock.patch.object(api_caller, '_call_api')
+    def test_api_invoked_correctly_for_mur(self, _call_api):
+        _call_api.return_value = {}
+        self.app.get(
+            '/legal/search/enforcement/',
+            data={
+                'mur_no': '6695',
+                'search': 'presidents',
+                'search_type': 'murs',
+            }
+        )
+        _call_api.assert_called_once_with('legal',
+                                          'search',
+                                          from_hit=0,
+                                          hits_returned=20,
+                                          murs_no='6995',
+                                          q='presidents',
+                                          type='murs')
```

Comment:
I'm doing TDD so I don't have to manually check the API call. Also to allow refactoring. 

What do you all think of the test name? `api_invoked_correctly_for_mur` — I like to name tests w/ a little more info, to describe what they're testing for. (As opposed to, say, `test_mur`.)